CHANGELOG.md

```diff
@@ -4,6 +4,37 @@
 
 - As of v1.3, support for `brokerList` is deprecated for our Kafka topic scaler and will be removed in v2.0 ([#632](https://github.com/kedacore/keda/issues/632))
 
+## v1.4
+
+### New
+
+- Extend RabbitMQ scaler to support count unacked messages([#700](https://github.com/kedacore/keda/pull/700))
+
+### Improvements
+
+- Fix scalers leaking ([#684](https://github.com/kedacore/keda/pull/684))
+- Provide installation YAML in the release ([#740](https://github.com/kedacore/keda/pull/740))
+- Handle nil pointers and empty arrays properly in Azure Monitor Scaler ([#680](https://github.com/kedacore/keda/pull/680))
```

Comment:
Maybe this is more clear from a function standpoint? Feels a bit too technical for a changelog.
```suggestion
- Improve Azure Monitor scaler to handle queries without metrics ([#680](https://github.com/kedacore/keda/pull/680))
```